start_time_intent_handler.py

```diff
@@ -159,12 +171,16 @@ def _MakeSpeechResponse(self, canonical_prayer, desired_prayer, prayer_time, cit
       return {'speech': 'Today, iftar is at %s in %s' % (prayer_time, city)}
     
     speech = ''
-    if city:
-      speech = 'The time for %s is %s in %s.' % (util.GetPronunciation(canonical_prayer), prayer_time, city)
-      display_text = 'The time for %s is %s in %s.' % (util.GetDisplayText(canonical_prayer), prayer_time, city)
+    if prayer_time:
+      if city:
+        speech = 'The time for %s is %s in %s.' % (util.GetPronunciation(canonical_prayer), prayer_time, city)
+        display_text = 'The time for %s is %s in %s.' % (util.GetDisplayText(canonical_prayer), prayer_time, city)
+      else:
+        speech = 'The time for %s is %s.' % (util.GetPronunciation(canonical_prayer), prayer_time)
+        display_text = 'The time for %s is %s.' % (util.GetDisplayText(canonical_prayer), prayer_time)
     else:
```

Comment:
this should only be called for location-based prayer time requests - we can define our own thing for masjid-based prayer time requests